File: src/main/java/net/thedragonteam/armorplus/potions/base/PotionBaseGood.java
/*
 * Copyright (c) TheDragonTeam 2016-2017.
 */

package net.thedragonteam.armorplus.potions.base;

/**
 * @author Sokratis Fotkatzikis - TheDragonTeam
 **/
public class PotionBaseGood extends PotionBase {

    public PotionBaseGood(int liquidColor, String name) {
        super(false, liquidColor, name);
        this.setBeneficial();
    }
}

File: src/main/java/net/thedragonteam/armorplus/potions/base/PotionBaseGood.kt
/*
 * Copyright (c) TheDragonTeam 2016-2017.
 */

package net.thedragonteam.armorplus.potions.base

/**
 * ArmorPlus created by sokratis12GR
 * - TheDragonTeam
 */
open class PotionBaseGood(liquidColorIn: Int, potionName: String) : PotionBase(false, liquidColorIn, potionName) {

    init {
        this.setBeneficial()
    }
}
